mlflow/utils/promptlab_utils.py

```diff
@@ -0,0 +1,14 @@
+import json
+
+
+def create_eval_results_file(prompt_parameters, model_input, model_output_parameters, model_output):
```

Comment:
```suggestion
def create_eval_results_json(prompt_parameters, model_input, model_output_parameters, model_output):
```

`create_eval_results_json` seems more accurate.